docs/source/using/profiles.adoc

```diff
@@ -8,52 +8,100 @@ include::variables.adoc[]
 
 toc::[]
 
-[[pofile-overview]]
+[[profile-overview]]
 == Overview
 
-{project} allows to create multiple instances of {project}.
-Each {project} instance can be created with different configurations and users can switch between different instances.
+A profile is an instance of the Minishift VM along with all of its configuration and state.
+The profile feature allows you to create and manage these isolated instances of Minishift.
 
-[[creating-profile]]
+Each {project} profile is created with its own configuration (memory, CPU, disk size, add-ons, and so on) and completely independent of other profiles.
+If you have common configuration options you want to apply to all profiles, you have to do this manually per profile.
+See xref:../using/profiles.adoc#setting-global-config-profiles[Setting Global Configuration For Profiles].
+
+The current profile being used is called the active profile.
+Unless the global `--profile` flag is used, {project} commands are executed against the active profile.
+You can determine the active profile by using the xref:../command-ref/minishift_profile_list.adoc#[`minishift profile list`] command. 
+You can execute single commands against a non-active profile, by using the `--profile` flag.
+Example:
+
+----
+$ minishift --profile my-profile console --url
+----
+
+On top of the `--profile` flag, there are xref:../command-ref/minishift_profile.adoc#[commands] for listing, deleting and setting the active profile.
+These commands are described in the following sections.
+
+[[creating-profiles]]
 == Creating Profiles
 
-You can create a new instance of {project} with xref:../command-ref/minishift_start.adoc#[`minishift start --profile`] command.
+A profile is created when any artifacts (example: configuration files) asociated with it gets created.
+There are two ways to create a new profile.
+
+==== Using the `--profile` flag
+
+When you run any {project} command with the `--profile` flag the profile gets created if it does not exist.
+
+Example:
+----
+$ minishift --profile PROFILE_NAME cofig set PROPERTY_NAME PROPERTY_VALUE
+----
+
+=== Using `profile set` command.
 
-NOTE: The default profile is `minishift`. It will be present by-default and user does not need to create it specifically.
+You can run the `profile set` command for a non existing profile to set it as the active profile.
+The atrifacts for the profile are automatically created when you subsequently run any `minishift` command.
```

Comment:
That's not really relevant for the user. Artifacts is ambiguous in this case. Also there is a typo 'atrifacts'